nighthawk/source/client/output.proto

```diff
@@ -26,4 +24,16 @@ message Statistic {
   google.protobuf.Duration mean = 3;
   google.protobuf.Duration pstdev = 4;
   repeated Percentile percentiles = 5;
-}
\ No newline at end of file
+}
+
+message Result {
+  string name = 1;
+  repeated Statistic statistics = 2;
+  repeated Counter counters = 3;
```

Comment:
Envoy has stats dumping protos, see https://github.com/envoyproxy/envoy/blob/master/api/envoy/admin/v2alpha/metrics.proto, should we reference these instead?

Reply:
Yes. I looked into that but I'm having trouble with the proto import. Adding this to the list to unblock here.

Reply:
Fair enough; would like to see this in the final NH but fine to leave for now.